workspaces/cicd-statistics/package.json

```diff
@@ -6,8 +6,14 @@
     "node": "18 || 20"
   },
   "scripts": {
+    "dev": "concurrently \"yarn start\" \"yarn start-backend\"",
+    "dev-next": "concurrently \"yarn start-next\" \"yarn start-backend\"",
+    "start": "yarn workspace app start",
+    "start-next": "yarn workspace app-next start",
+    "start-backend": "yarn workspace backend start",
+    "build:backend": "yarn workspace backend build",
     "tsc": "tsc",
-    "tsc:full": "tsc --skipLibCheck false --incremental false",
+    "tsc:full": "tsc --skipLibCheck true --incremental false",
```

Comment:
Not sure why there change here, the way it was it what you'll see in all the root `package.json` files in each workspace.

Reply:
There looks to be an issue with the `@material-ui/pickers` which is why this got flipped. I think it's safe to leave for now as we will be moving away from this package: https://github.com/backstage/community-plugins/issues/457